src/NimBLERemoteCharacteristic.cpp

```diff
@@ -463,7 +482,7 @@ bool NimBLERemoteCharacteristic::registerForNotify(notify_callback notifyCallbac
 /**
  * @brief Delete the descriptors in the descriptor vector.
  * We maintain a vector called m_descriptorVector that contains pointers to BLERemoteDescriptors
- * object references. Since we allocated these in this class, we are also responsible for deleteing
+ * object references. Since we allocated these in this class, we are also responsible for deleting
```

Comment:
You're good a finding these, I'm surprised you haven't seen the really bad one in client yet 😄.

Reply:
Lol just by coincidence, and English isn't even my native language :smile: